itest/litd_custom_channels_test.go

```diff
@@ -227,13 +227,13 @@ func testCustomChannels(_ context.Context, net *NetworkHarness,
 	charlieAssetBalance -= keySendAmount
 	daveAssetBalance += keySendAmount
 
-	// We should be able to send the 100 assets back immediately, because
+	// We should be able to send the 1000 assets back immediately, because
 	// there is enough on-chain balance on Dave's side to be able to create
-	// an HTLC.
-	sendAssetKeySendPayment(
-		t.t, dave, charlie, keySendAmount, assetID, fn.None[int64](),
+	// an HTLC. We use an invoice to execute another code path.
+	invoiceResp := createAssetInvoice(
+		t.t, dave, charlie, keySendAmount, assetID,
 	)
-	logBalance(t.t, nodes, assetID, "after keysend back")
+	payInvoiceWithAssets(t.t, dave, charlie, invoiceResp, assetID)
```

Comment:
so prior to fix the sender would crash here saying there's no available path?

Reply:
Yes, a NO_ROUTE error would be thrown here.